### PR TITLE
Feature/select heatmap

### PR DESCRIPTION
**Pull request by danielbabiak**

### CODE REVIEW

File: src/Spectre.AngularClient/src/app/preparations/preparation/preparation.component.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<p>
+  preparation {{id}} works!
+</p>
+<label>Mz:</label>
+
+<input [(ngModel)]="mzValue" type="number"/>
+
+<md-slider class="slider" [(ngModel)]="currentChannelId" showTicks="true" min="0" max={{mzLenth}} step="1" (input)="onInputChannelId($event)"
```
Comment (Sebaxp95) [inline src/Spectre.AngularClient/src/app/preparations/preparation/preparation.component.html]: spelling

File: src/Spectre.AngularClient/src/app/preparations/preparation/preparation.component.ts [hunk 2]
```diff
@@ -38,6 +37,10 @@ export class PreparationComponent implements OnInit {
   public id: number;
   public heatmapData: any;
   public spectrumData: any;
+  public mzLenth: number;
```
Comment (Sebaxp95) [inline src/Spectre.AngularClient/src/app/preparations/preparation/preparation.component.ts]: spelling

File: src/Spectre/Models/Msi/Preparation.cs [hunk 3]
```diff
@@ -53,6 +53,15 @@ public class Preparation
         public int SpectraNumber { get; private set; }
 
         /// <summary>
+        /// Gets the number of chanelIds.
+        /// </summary>
+        /// <value>
+        /// Number of chanelIds.
+        /// </value>
+        [DataMember]
+        public int ChanelIdAvailableNumber { get; private set; }
```
Comment (Sebaxp95) [inline src/Spectre/Models/Msi/Preparation.cs]: Unnecessary

Comment (gmrukwa) [inline src/Spectre/Models/Msi/Preparation.cs]: I think we will have to redesign everything a little bit soon, so let it be.

**Comment by gmrukwa** [comment]
Good job with migration to new version of Angular!

**Comment by gmrukwa** [comment]
[Display any heatmap](https://trello.com/c/Qg0R7Xsd/136-display-any-heatmap)

**Pull request merged**